html/anushka_sharmila.html

```diff
@@ -0,0 +1,87 @@
+images/portraits/
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Anushka and Sharmila</title>
+  <link rel="preconnect" href="https://fonts.gstatic.com" />
```

Comment:
See if there's a way to reuse the same HTML code carried across each file, excluding the `<title>` tag.